backend/deploy/templates/backend.deployment.yaml

```diff
@@ -65,5 +67,11 @@ spec:
           runAsNonRoot: true
           seccompProfile:
             type: RuntimeDefault
+        readinessProbe:
+          httpGet:
+            path: /healthz
+            port: 8083
+          initialDelaySeconds: 5
+          periodSeconds: 10            
```

Comment:
It looks like there's trailing spaces on this line.  That's probably what's tripping the CI failures.